utils/src/shell_wrapper.rs

```diff
@@ -100,16 +110,24 @@ fn real_main() -> anyhow::Result<()> {
 }
 
 fn main() {
-    if let Err(err) = JournalLog::new()
-        .context("When initializing journal logger")
-        .and_then(|logger| {
-            logger
-                .with_syslog_identifier("shell-wrapper".to_string())
-                .install()
-                .context("When installing journal logger")
-        })
-    {
-        warn!("Error while setting up journal logger: {:?}", err);
+    if nix::unistd::geteuid().is_root() {
+        // Journal isn't available during early boot
```

Comment:
Wouldn't this make all root shells log to kmsg? It's not awful, but suboptimal IMO. Maybe we should just use kmsg as a fallback for when the journal logger setup fails?

Reply:
It does, but kmsg also appears in the journal, so I didn't think about it too much. But it's probably better to just use kmsg as a fallback so we don't clutter kmsg even more. I'll change that